test/services/role/services/userRoles.test.js

```diff
@@ -0,0 +1,144 @@
+const { expect } = require('chai');
+const app = require('../../../../src/app');
+const testObjects = require('../../helpers/testObjects')(app);
+
+const userRoles = app.service('/roles/user');
+
+describe('userRoles', async () => {
+	let server;
```

Comment:
add the following setup into a before and reuse the code from before

Reply:
I'm sorry, but I don't understand. Why should I do this if I only use it once to initiate the data needed for the tests? You probably mean that I use the same data for testing for userPermissions.test.js but then I don't see how it would work.